nomad/structs/structs.go

```diff
@@ -5058,11 +5060,27 @@ type JobSummary struct {
 func (js *JobSummary) Copy() *JobSummary {
 	newJobSummary := new(JobSummary)
 	*newJobSummary = *js
+
 	newTGSummary := make(map[string]TaskGroupSummary, len(js.Summary))
 	for k, v := range js.Summary {
 		newTGSummary[k] = v
 	}
+
+	newVTGSummary := make(map[uint64]VersionedSummary, len(js.VersionedSummary))
+	for k, v := range js.VersionedSummary {
+		uk := uint64(k)
```

Comment:
Would be thrilled to find out there was a better way to do this!

Reply:
nvm — this was from a version where I was trying to set the key to a string well outside of serialization.